namespace py neteng.fboss.bcm_config
namespace py.asyncio neteng.fboss.asyncio.bcm_config
namespace cpp2 facebook.fboss.bcm

struct BcmConfig {
  1: map<string, string> config
}
